_docker/drupal/drupal-filesystem/web/config/sync/assembly.assembly_type.content_image.yml

```diff
@@ -3,7 +3,7 @@ langcode: en
 status: true
 dependencies: {  }
 id: content_image
-label: 'Content Image'
-description: 'Image that breaks the grid width to the right.'
+label: Image
+description: 'Image must be uploaded locally'
```

Comment:
@jordanpagewhite Technically in your spreadsheet it says Image`s` must be uploaded locally. Not sure if that's too big a deal?